app/views/applications/show.html.erb

```diff
@@ -16,6 +16,18 @@
     <%= form.submit "Search" %>
   <% end %>
   <% if @search_results %>
-      <p>Search Results: <%= @search_results.join(" | ") %></p>
+      <p>Search Results:</p>
+    <% @search_results.each do |pet| %>
```

Comment:
i think we should move @search_results outside of the conditional @app.status == "In Progress" as status isn't required to search. So line 9 & 18 should be on the same indentation plane.

Reply:
This logic checks that only applications that are "In Progress" show the search bar. If the app is completed or in review it shouldn't be able to add a new pet